quantum/painter/qp_draw_text.c

```diff
@@ -1,4 +1,5 @@
 // Copyright 2021 Nick Brassel (@tzarc)
+// Copyright 2023 Pablo Martinez (@elpekenin) <elpekenin@elpekenin.dev>
```

Comment:
You're not really going to be able to claim copyright on such a minor change
```suggestion
```

Reply:
> You're not really going to be able to claim copyright on such a minor change

Makes  complete sense and I actually dont mind being on the header or not. For future reference, is there any rule of thumb to know when it should be added?

Reply:
Realistically for this project and the submissions we get, almost never. There is a case when the majority of a file has been rewritten.